rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -605,33 +644,36 @@ rocalResize(
     return output;
 }
 
-RocalImage ROCAL_API_CALL
+RocalTensor ROCAL_API_CALL
 rocalBrightness(
         RocalContext p_context,
-        RocalImage p_input,
+        RocalTensor p_input,
         bool is_output,
         RocalFloatParam p_alpha,
-        RocalFloatParam p_beta)
-{
-    Image* output = nullptr;
+        RocalFloatParam p_beta,
+        RocalTensorLayout rocal_tensor_output_layout,
+        RocalTensorOutputType rocal_tensor_output_datatype) {
+    rocalTensor* output = nullptr;
     if ((p_context == nullptr) || (p_input == nullptr)) {
-        ERR("Invalid ROCAL context or invalid input image")
+        ERR("Invalid ROCAL context or invalid input tensor")
         return output;
     }
 
     auto context = static_cast<Context*>(p_context);
-    auto input = static_cast<Image*>(p_input);
+    auto input = static_cast<rocalTensor*>(p_input);
     auto alpha = static_cast<FloatParam*>(p_alpha);
     auto beta = static_cast<FloatParam*>(p_beta);
-    try
-    {
-
-        output = context->master_graph->create_image(input->info(), is_output);
-
+    try {
+        RocalTensorlayout op_tensorLayout;
+        RocalTensorDataType op_tensorDataType;
+        get_rocal_tensor_layout(rocal_tensor_output_layout, op_tensorLayout);
```

Comment:
if the output layout and type is the same as input, please get rid of the unnecessary functions and use the input layout and type directly

Reply:
This has been removed in recent commits